src/Keboola/GoogleAnalyticsExtractor/GoogleAnalytics/Client.php

```diff
@@ -228,10 +228,7 @@ private function processResponse($response, $query)
 
     private function processResponseMCF($response, $query)
     {
-        if (empty($response['rows'])) {
-            return [];
-        }
-        $rows = $response['rows'];
+        $rows = empty($response['rows']) ? [] : $response['rows'];
```

Comment:
fix: Ked v odpovedi nie su "rows" znamena to, ze nie su ziadne vysledky pre pozadovany date range, ale tado metoda by mala vratit ocakavany "objekt" a nie prazdne pole.